src/user/user.ts

```diff
@@ -120,7 +120,7 @@ export class User extends Observable<SubscriberUser> {
       throw new Error('AccessToken is missing or has expired')
     }
     // get account specified in access token
-    const account = this?._accountName
+    const account = this?._accountName || this?._accessTokenHelper.accountName
```

Comment:
@traylewin Sometimes `this._accountName` is null but it's present in `this?._accessTokenHelper.accountName`
This change fixes that issue.

Reply:
This doesn't solve the root of the problem it just hacks around it and adds more future confusion.
I would guess the solution is to add code in onUpdateAccessTokenHelper() to update this.accountName from accessTokenHelper.accountName. revert this code and add the fix there like...
  private onUpdateAccessTokenHelper = (newAccessTokenHelper: AccessTokenHelper) => {
    this._accountName = newAccessTokenHelper.accountName
    super.callSubscribers()
  }